### PR TITLE
add Search ui button improvement & add time conflict acknowledge

### PR DESCRIPTION
**Pull request by Poroma**

### CODE REVIEW

**Comment by heatsink** [comment]
Nice work. I'm tacking on this change to the scheduling conflict text your commit:
screen shot 2018-06-16 at 7 33 54 pm
Also wrote a couple of tests.

File: lib/generic/genericAlert.dart [hunk 1]
```diff
@@ -31,4 +32,29 @@ class GenericAlert {
           );
         });
   }
+
```
Comment (RedTo) [inline lib/generic/genericAlert.dart]: Please hit Control + Alt + L to format the code in this file.

File: lib/generic/genericIcon.dart [hunk 2]
```diff
@@ -49,6 +51,16 @@ class GenericIcon {
     );
   }
 
+  static Icon buildGenericSearcgIcon(bool isActive) {
```
Comment (RedTo) [inline lib/generic/genericIcon.dart]: Should be renamed to buildGenericSearchIcon

File: lib/views/schedule.dart [hunk 3]
```diff
@@ -26,6 +27,7 @@ class Schedule extends StatelessWidget {
     return new ListView(
       children: children,
     );
+
```
Comment (RedTo) [inline lib/views/schedule.dart]: Please also format the code in this file.
 - remove empty lines
 - make spaces like in other files etc

-> Ctrl + Alt + L (should be standard in IntelliJ and Android Studio)

**Pull request merged**